public/scripts/services/form.js

```diff
@@ -3,9 +3,9 @@
 
     window.ls.container.set('form', function () {
 
-        function cast(value, to) {
+        function cast(value, from, to,) {
```

Comment:
What exactly was the problem here? This is a risky change as it affects most forms on the console, let's validate carefully.

Reply:
The problem is when binding an array to a hidden input, it is being bound as comma separated string, so when passed to `cast`, it is cast to a single element array instead of splitting into individual elements

E.g given

```javascript
permissions = ['read("any")', 'create("any")'];
```

Would bind as:

`'read("any"),create("any")'`

Which would cast to:

`['read("any"),create("any")']`

Instead of the expected

`['read("any")', 'create("any")']`

The change should only have any effect when both `data-cast-to="array"` and `data-cast-from="csv"` are set on the same element